internal/pkg/docker/dockerfile/dockerfile.go

```
@@ -62,50 +70,59 @@ type HealthCheck struct {
 
 // Dockerfile represents a parsed Dockerfile.
 type Dockerfile struct {
-	ExposedPorts []portConfig
-	HealthCheck  *HealthCheck
+	exposedPorts []Port
+	healthCheck  *HealthCheck
 	parsed       bool
 	path         string
 
 	fs afero.Fs
 }
 
-// NewDockerfile returns an empty Dockerfile.
-func NewDockerfile(fs afero.Fs, path string) *Dockerfile {
+// New returns an empty Dockerfile.
+func New(fs afero.Fs, path string) *Dockerfile {
 	return &Dockerfile{
-		ExposedPorts: []portConfig{},
-		HealthCheck:  nil,
+		exposedPorts: []Port{},
+		healthCheck:  nil,
 		fs:           fs,
 		path:         path,
 		parsed:       false,
```

Probably can be just
```suggestion
		fs:           fs,
		path:         path,
```

---

Done!